README.md

```diff
@@ -245,6 +245,39 @@ Travis already have a provider for deploying to bintray, but it is extremely
 inconvenient as it requires to produce one json file per file to upload.
 
 
+Codecov
+-------
+
+Beaver includes a convenient wrapper to send [codecov.io](https://codecov.io/)
+reports. Just make sure the project was already built with coverage support and
+then use something like this in the `.travis.yml` file:
+
+```yml
+after_success: BEAVER_CODECOV_REPORTS="*.lst" beaver codecov [OPTIONS]
+```
+
+The `[OPTIONS]` are forwarded directly to the
+[codecov-bash](https://github.com/codecov/codecov-bash), but this script will
+run in a confined docker instance where only the coverage reports are available,
+plus the original source code as it was in Git (not the, possibly dirtly, build
+workspace).
+
+Only `TRAVIS*` and `CODECOV_*` environment variables will be automantically
+passed to the docker container. If you want to pass any other environment
+variables use `BEAVER_DOCKER_VARS` as usual.
+
+The reports location must be passed explicitly to the script via the
+`BEAVER_CODECOV_REPORTS` environment variable. Glob patterns (like `*.lst`) and
```

Comment:
Just wondering why it is required to set the location of the report. This can already be specified passed to `codecov.bash` using `-f`. https://github.com/leandro-lucarella-sociomantic/beaver/blob/2c4d96cd009595b759eb713e80115856663eeea1/bin/codecov-bash#L89

And if the `[OPTIONS]` are parsed directly to the codecov.bash script the requirement of the env var **BEAVER_CODECOV_REPORTS** seems redundant.

Reply:
> Just wondering why it is required to set the location of the report.

Because the reports need to be copied to the sandbox directory first.

Reply:
> Because the reports need to be copied to the sandbox directory first.

Hmm oki. But for `libdrizzle-redux` where will the report be copied to? 
In https://github.com/sociomantic-tsunami/libdrizzle-redux/pull/271 the upload step isn't performed in a `docker` container but in the the native `travis` build environment, with the report file being uploaded directly from the cache-dir

Reply:
> Hmm oki. But for libdrizzle-redux where will the report be copied to?

You don't care, that's taken care by beaver. Just tell beaver where to find the reports :)

Reply:
> In sociomantic-tsunami/libdrizzle-redux#271 the upload step isn't performed in a docker container but in the the native travis build environment, with the report file being uploaded directly from the cache-dir

And when you switch to beaver, beaver will take care to build a sandbox for you, copy the reports to tell it to copy and run the `codecov` script in that sandbox. That's the whole point of using beaver :)

Reply:
> And when you switch to beaver, beaver will take care to build a sandbox for you,

Oki, I will let beaver dig in  

![image](https://user-images.githubusercontent.com/13032446/35828821-328c59d2-0ac1-11e8-8d5f-79393d87d470.png)

Reply:
😱